tests/virt_autotest/prepare_transactional_server.pm

```diff
@@ -86,7 +86,7 @@ sub install_additional_pkgs {
         zypper_call("--gpg-auto-import-keys refresh");
         save_screenshot;
 
-        my $cmd = "install --no-allow-downgrade --no-allow-name-change --no-allow-vendor-change";
+        my $cmd = "install --no-allow-downgrade --no-allow-vendor-change";
```

Comment:
See my comment [1](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20498#issuecomment-2443721359) and [2](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20498#issuecomment-2448853946).

Reply:
Another solution is that handling the package installation in kubevirt test code.

Reply:
Yes. I prefer you handle in your own test. So other tests will not have this risky code.